articles/aks/concepts-identity.md

```diff
@@ -141,6 +141,22 @@ AKS provides the following four built-in roles. They are similar to the [Kuberne
+
+The Role Grant referred to in the second column is the AAD role grant shown on the Access Control Tab of the Portal, while the Cluster Admin AAD Group is shown on the Configuration tab in the Portal (or with parameter name `--aad-admin-group-object-ids` in the Azure CLI).
+
+| Description        | Role grants| Cluster admin AAD group(s) | When to use |
+| -------------------|------------|----------------------------|-------------|
+| Legacy admin login| **Azure Kubernetes Admin Role**. This role allows `az aks get-credentials` to be used with the `--admin` flag, which downloads a [legacy (non-AAD) cluster admin certificate](https://docs.microsoft.com/en-us/azure/aks/control-kubeconfig-access) into the user's `.kube/config`. This is the only purpose of "Azure Kubernetes Admin Role".|n/a|If you're permanently blocked by not having access to a valid Azure AD group with access to your cluster| 
+| AAD with manual (Cluster)RoleBindings| **Azure Kubernetes User Role**. The "User" role allows `az aks get-credentials` to be used without the `--admin` flag. (This is the only purpose of "Azure Kubernetes User Role".) The result, on an AAD-enabled cluster, is the download of [an empty entry](https://docs.microsoft.com/en-us/azure/aks/control-kubeconfig-access) into `.kube/config`, which triggers browser-based authentication when it's first used by `kubectl`.| User is not in any of these groups. Because the user is not in any Cluster Admin groups, their rights will be controlled entirely by any RoleBindings or ClusterRoleBindings that have been set up by cluster admins. The (Cluster)RoleBindings [nominate AAD users or AAD groups](https://docs.microsoft.com/en-us/azure/aks/azure-ad-rbac) as their `subjects`. If no such bindings have been set up, the user will not be able to excute any `kubectl` commands.|If you want fine-grained access control, and you're not using Azure RBAC for Kubernetes Authorization. Note that the user who sets up the bindings must log in by one of the other methods listed in this table.|
+| AAD by member of admin group| same as above|User is a member of one of the groups listed here. AKS automatically generates a ClusterRoleBinding that binds all of the listed groups to the cluster-admin Kubernetes role. So users in these groups can run all `kubectl` commands|If you want to conveniently grant users full admin rights, and are _not_ using Azure RBAC for Kubernetes authorization|
+| AAD with Azure RBAC for Kubernetes Authorization|Two roles: firstly **Azure Kubernetes User Role** (as above). Secondly one of the "Azure Kubernetes Service **RBAC**..." roles listed above, or your own custom alternative|The admin roles field on the Configuration tab is irrelevant when Azure RBAC for Kubernetes Authorization is enabled|You are using Azure RBAC for Kubernetes authorization. This approach gives you fine-grained control, without the need to set up RoleBindings or ClusterRoleBindings|
```

Comment:
@miwithro @JohnRusk I think this points out a fact that we are missing the concept of "control plane" and "data plane" in this doc. that is required to setup the reason why there are two roles needed.

Reply:
There is a section in the middle of the doc, which explains the reason why two roels are used.  The heading is "Azure role-based access control (Azure RBAC)" and the key text is introduced by the sentence 
> There are two levels of access needed to fully operate an AKS cluster:
Does that do what you want?  Should it perhaps be made more disoverable, by moving it to the start of the document???

Reply:
Yeah. I think that should be explained upfront?

Reply:
@palma21 Do you want to do a separate PR to explain that upfront, or would you prefer I have a go at that change too?  In this PR or a separate one?